src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -44,7 +44,7 @@ public void testIsFull() {
     @Test
     public void testPeekaboo() {
         DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-        final int EMPTY_STACK_VALUE = -1;
+        final int EMPTY_STACK_VALUE = 0;

```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).